src/calibre/gui2/viewer/main.py

```diff
@@ -759,6 +787,19 @@ def update_indexing_state(self, anchor_positions=None):
                                 self.toc_model.currently_viewed_entry, entry,
                                 pgns[2])
                         self.toc_clicked(entry.index(), force=True)
+        #if hasattr(self, 'current_page'):
```

Comment:
not a fan of having commented code - likely to be stale and can't be automatically be checked.

Reply:
Again, this PR is still under development, and I didn't want to have to mess with git to revert changes.